create-prs-to-update-vcs-repositories/create_prs_to_update_vcs_repositories.py

```diff
@@ -353,11 +353,19 @@ def main(args: argparse.Namespace) -> None:
             # Switch back to base branch
             repo.heads[args.base_branch].checkout()
 
+            # Compose a PR body
+            pr_body = f'''## Description
+This PR updates the version of the repository {repo_name} in autoware.repos.
+
+## What's changed
+You can see the changes in its [git diff](https://github.com/{repo_name}/compare/{current_version}...{latest_tag}).
```

Comment:
I prefer the direct link instead.

```suggestion
You can see the changes in https://github.com/{repo_name}/compare/{current_version}...{latest_tag}.
```